### ISSUE TITLE
pip install is missing mapfile.g

### ISSUE THREAD

**Issue by ARSimmons** [geographika/mappyfile#3]
Last week I installed 'mappyfile' from pip and ran a ' mappyfile.load(file)' command. The strack trace threw this back at me. I checked that directory...the error doesn't lie. Indeed there is no 'mapfile.g'. 

Can someone verify the pip install package is ok....and if it isn't...is the .whl ok?

IOError: [Errno 2] No such file or directory: 'C:\\Users\\ASimmons\\.virtualenvs\\gis_test\\lib\\site-packages\\mappyfile\\mapfile.g'

**Comment by erezsh** [comment]
I'm guessing it has something to do with the following lines in setup.py?
      package_data = {
        '': ['*.g']
      },

**Comment by ARSimmons** [comment]
I'm not following what you mean. Here is the full stack trace though.

Traceback (most recent call last):
  File "E:/March_2017_Hackathon/Mappyfile/find_all_regex.py", line 29, in <module>
    parsed_file = mappyfile.load(file)
  File "C:\Users\ASimmons\.virtualenvs\gis_test\lib\site-packages\mappyfile\utils.py", line 8, in load
    p = Parser(cwd=cwd)
  File "C:\Users\ASimmons\.virtualenvs\gis_test\lib\site-packages\mappyfile\parser.py", line 13, in __init__
    grammar_text = open(gf).read()
IOError: [Errno 2] No such file or directory: 'C:\\Users\\ASimmons\\.virtualenvs\\gis_test\\lib\\site-packages\\mappyfile\\mapfile.g'
>>>

**Comment by erezsh** [comment]
Sorry, I mean it definitely looks like a bug. I was just making a suggestion to @geographika .

**Comment by geographika** [comment]
@ARSimmons - this was an issue in a previous release (0.1.2), but was fixed in 0.1.3 a couple of weeks ago in pypi - https://pypi.python.org/pypi/mappyfile/

The following works fine for me on Windows, and correctly includes C:\VirtualEnvs\mappyfile_test\Lib\site-packages\mappyfile\mapfile.g

```
md C:\VirtualEnvs\mappyfile_test
cd C:\VirtualEnvs
C:\Python27\Scripts\virtualenv.exe mappyfile_test

cd mappyfile_test\scripts
activate

pip install -U pip
pip install mappyfile

python 
import mappyfile
```

Is there any chance you have a cached version of the package?
@erezsh - the [*.g] line fixed the issue from 0.1.2 - I got it from plyplus setup.py :)

**Comment by ARSimmons** [comment]
@geographika 

You are right...I happened to still have 0.1.2 on my machine. I uninstalled and upgraded to 0.1.3. Closing.


 Successfully uninstalled mappyfile-0.1.2

gis_test) C:\Users\ASimmons\.virtualenvs\gis_test\Scripts>pip install mappyfile

  c:\users\asimmons\.virtualenvs\gis_test\lib\site-packages\mappyfile\utils.py
  c:\users\asimmons\.virtualenvs\gis_test\lib\site-packages\mappyfile\utils.pyc
Proceed (y/n)? y
  Successfully uninstalled mappyfile-0.1.2

(gis_test) C:\Users\ASimmons\.virtualenvs\gis_test\Scripts>pip install mappyfile

...
  Downloading mappyfile-0.1.3-py2.py3-none-any.whl
Requirement already satisfied (use --upgrade to upgrade): plyplus in c:\users\as
immons\.virtualenvs\gis_test\lib\site-packages (from mappyfile)
Requirement already satisfied (use --upgrade to upgrade): ply in c:\users\asimmo
ns\.virtualenvs\gis_test\lib\site-packages (from plyplus->mappyfile)
Installing collected packages: mappyfile
Successfully installed mappyfile-0.1.3
